src/domain/settlementWindow/index.js

```diff
@@ -58,15 +59,23 @@ module.exports = {
 
   getByParams: async function (params, enums) {
     // 4 filters - at least one should be used
-    if (hasFilters(params.query) && Object.keys(params.query).length < 5) {
+    if (hasFilters(params.query) && Object.keys(params.query).length < 6) {
```

Comment:
@lazolalucas, you need to update the following method **SettlementWindowModel.getByParams(params, enums)** to take into account the newly introduced param - currency. Currently for the following request:
`curl -X GET 'http://127.0.0.1:3007/v1/settlementWindows?currency=TZS'`, I'm getting response:
```
[
    {
        "settlementWindowId": 2,
        "state": "CLOSED",
        "reason": "testing",
        "createdDate": "2020-02-24T15:01:38.000Z",
        "changedDate": "2020-02-24T17:55:57.000Z",
        "content": [
            {
                "id": 4,
                "state": "CLOSED",
                "ledgerAccountType": "POSITION",
                "currencyId": "USD",
                "createdDate": "2020-02-24T17:55:57.000Z",
                "changedDate": "2020-02-24T17:55:57.000Z",
                "settlementId": null
            }
        ]
    },
    {
        "settlementWindowId": 3,
        "state": "OPEN",
        "reason": "testing",
        "createdDate": "2020-02-24T17:55:57.000Z",
        "changedDate": "2020-02-24T17:55:57.000Z",
        "content": []
    },
    {
        "settlementWindowId": 1,
        "state": "SETTLED",
        "reason": "All settlement window content is SETTLED",
        "createdDate": "2020-02-24T13:00:39.000Z",
        "changedDate": "2020-02-24T13:01:38.000Z",
        "content": [
            {
                "id": 1,
                "state": "SETTLED",
                "ledgerAccountType": "POSITION",
                "currencyId": "USD",
                "createdDate": "2020-02-24T15:01:38.000Z",
                "changedDate": "2020-02-24T13:01:38.000Z",
                "settlementId": 1
            },
            {
                "id": 2,
                "state": "SETTLED",
                "ledgerAccountType": "POSITION",
                "currencyId": "TZS",
                "createdDate": "2020-02-24T15:01:38.000Z",
                "changedDate": "2020-02-24T13:01:38.000Z",
                "settlementId": 2
            }
        ]
    }
]
```
I'd expect to get only **settlementWindowId 1** in the above array of objects.